concrete/blocks/page_attribute_display/form.php

```diff
@@ -9,7 +9,7 @@
 ?>
 <div class="ccm-tab-content" id="ccm-tab-content-options">
     <div class="form-group">
-        <label class="control-label"><?php  echo t('Display property with formatting')?></label>
+        <label class="control-label"><?php  echo t('Display Property with Formatting')?></label>
```

Comment:
> Are these distinct translation strings?

@KorvinSzanto Yes, they are (but Community Translation should be smart enough to suggest translators the translation of the previous string).